image-meta-tag/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+    name: image-meta-tag
+    version: "0.4"
```

Comment:
For consistency and good practice, this should be changed to match the version number exactly (0.4.1)